logdetective/server/models.py

```diff
@@ -323,6 +331,63 @@ def __init__(self, data: Optional[dict] = None):
             self.instances[instance.url] = instance
 
 
+class KojiInstanceConfig(BaseModel):
+    """Model for Koji configuration of logdetective server."""
+
+    name: str = ""
+    xmlrpc_url: str = ""
+    max_artifact_size: int = 300 * 1024 * 1024
+    tokens: List[str] = []
+
+    _conn: Optional[koji.ClientSession] = None
+
+    def __init__(self, name: str, data: Optional[dict] = None):
+        super().__init__()
+
+        self.name = name
+        if data is None:
+            # Set some reasonable defaults
+            self.xmlrpc_url = "https://koji.fedoraproject.org/kojihub"
+            self.tokens = []
+            self.max_artifact_size = 1024 * 1024
+            return
+
+        self.xmlrpc_url = data.get(
+            "xmlrpc_url", "https://koji.fedoraproject.org/kojihub"
+        )
+        self.max_artifact_size = int(data.get("max_artifact_size")) * 1024 * 1024
```

Comment:
Default should be set to some reasonable value, probably `300` as it is on 339

Reply:
That's a typo. It was indeed supposed to be 300 there.